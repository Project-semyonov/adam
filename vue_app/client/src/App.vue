<template>
  <div id="app">
    <router-view/>
  </div>
</template>


<script>
export default {
  name: 'App',
};
</script>

<style>
.header{
  text-align: center;
}
.temp-table{
  text-align: center;
  table-layout: fixed;
  margin: auto;
  width: 20%;
  color: azure;
}
.temp-table-header{
  background-image: linear-gradient(to bottom right, rgba(110, 3, 83, 0.6),rgba(173, 101, 6, 0.6));
}
.temp-alert{
  padding: 10px;
  background-image: linear-gradient(to bottom right,
  rgba(110, 3, 83, 0.6),rgba(173, 101, 6, 0.6));
}
tr:nth-child(even) {
  background-image: linear-gradient(to bottom right, rgba(110, 3, 83, 0.6),rgba(173, 101, 6, 0.6));
}
tr:nth-child(odd) {
  background-image: linear-gradient(to bottom right, rgba(173, 101, 6, 0.6),rgba(110, 3, 83, 0.6));
}
html,body, #app{
  height: 100%;
}
#app {
  font-family: 'Avenir', Helvetica, Arial, sans-serif;
  -webkit-font-smoothing: antialiased;
  -moz-osx-font-smoothing: grayscale;
  background:url('assets/img/home-bg.jpg');
  -webkit-background-size: cover;
  -moz-background-size: cover;
  -o-background-size: cover;
  background-size: cover;
  color: orange;
  text-align: center;
}
</style>
