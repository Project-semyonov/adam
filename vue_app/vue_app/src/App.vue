<template>
  <div id="app">
    <div class="header">
      <div class="home-button">
        <router-link to="/home" tag="button" class="btn btn-danger">Homepage</router-link>
      </div>
      <router-link to="/camera" tag="button" class="btn btn-outline-warning"
        >Camera</router-link>
      <router-link to="/temp" tag="button" class="btn btn-outline-warning"
        >Temp Sensor</router-link>
      <div class="github-button">
        <a href="https://github.com/Project-semyonov/adam" tag="button"
        class="btn btn-outline-success">Github Repo</a>
      </div>
    </div>
    <router-view/>
  </div>
</template>

<style>
.header{
  text-align: center;
}
.home-button{
  float: left;
}
.github-button{
  float: right;
}
.temp-table{
  text-align: center;
  table-layout: fixed;
  margin: auto;
  width: 20%;
  color: azure;
}
.home-temp-table-header{
  text-align: left;
  table-layout: fixed;
  margin: auto;
  width: 20%;
  color: azure;
}
.temp-table-header{
  background-image: linear-gradient(to bottom right, rgba(110, 3, 83, 0.6),rgba(173, 101, 6, 0.6));
}
.temp-alert{
  padding: 10px;
  background-image: linear-gradient(to bottom right,
  rgba(110, 3, 83, 0.6),rgba(173, 101, 6, 0.6));
}
tr:nth-child(even) {
  background-image: linear-gradient(to bottom right, rgba(110, 3, 83, 0.6),rgba(173, 101, 6, 0.6));
}
tr:nth-child(odd) {
  background-image: linear-gradient(to bottom right, rgba(173, 101, 6, 0.6),rgba(110, 3, 83, 0.6));
}
html,body, #app{
  height: 100%;
}
#app {
  font-family: 'Avenir', Helvetica, Arial, sans-serif;
  -webkit-font-smoothing: antialiased;
  -moz-osx-font-smoothing: grayscale;
  background:url('assets/home-bg.jpg');
  -webkit-background-size: cover;
  -moz-background-size: cover;
  -o-background-size: cover;
  background-size: cover;
  color: orange;
  text-align: center;
}
</style>
