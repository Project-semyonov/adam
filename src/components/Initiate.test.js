// import App from '../components/App/App.vue'
import initStoryshots from '@storybook/addon-storyshots'
import App from '../components/App/App.vue'
initStoryshots()
console.log(App)

test('INDEX', () => {
  console.log('haha')

  // const vue = createLocalVue()
  // const app = mount(App, { vue })

  // expect(app.classes()).toContain('center-content')
  expect(5).toEqual(5)
})
