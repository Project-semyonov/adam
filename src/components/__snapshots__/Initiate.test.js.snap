// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Storyshots HelloComponent story as a component 1`] = `
<div>
  <h1>
    Hello Adam!
  </h1>
   
  <h2
    class="wave"
  >
    H2 Is here
  </h2>
</div>
`;
